com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/ShaderUtilities.cs

```diff
@@ -400,5 +413,25 @@ private static string RemoveWhitespace(string input)
         {
             return new string(input.ToCharArray().Where(c => !Char.IsWhiteSpace(c)).ToArray());
         }
+
+        /// <summary>
+        /// Checks if string is a Header attribute
+        /// </summary>
+        private static bool isHeaderAttribute(string a)
```

Comment:
nit, normally we pascal casing for methods.

```suggestion
        private static bool IsHeaderAttribute(string a)
```